frontend/src/app/shared/ui/components/dialog/dialog.component.html

```
@@ -1,19 +1,13 @@
-﻿<dialog #dialog class="modal">
+﻿<dialog #dialog class="modal" [open]="isOpen()">
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)